packages/elements-dev-portal/src/components/NodeContent/NodeContent.tsx

```diff
@@ -127,11 +127,8 @@ const LinkComponent: CustomComponentMapping['a'] = ({ children, href }) => {
     );
 
     if (edge) {
-      return (
-        <Link to={edge.slug} hash={hash}>
-          {children}
-        </Link>
-      );
+      console.log('Edge Found ', edge);
```

Comment:
Let's not commit that. Though this is my favourite way of debugging 😉

Reply:
removed :)